eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -184,6 +184,10 @@ jobs:
             ServiceDirectory: ${{ parameters.ServiceDirectory }}
             TestPipeline: ${{ parameters.TestPipeline }}
 
+      - pwsh: |
+          Write-Host "##vso[task.setvariable variable=DOTNET_ROLL_FORWARD]Major"
+        displayName: Set DOTNET_ROLL_FORWARD to Major globally
```

Comment:
This should be done with eng/common changes, not globally set here.

Reply:
https://github.com/Azure/azure-sdk-tools/pull/8373

Reply:
With https://github.com/Azure/azure-sdk-tools/pull/8373/ and the sync https://github.com/Azure/azure-sdk-for-cpp/pull/5699 merged, do we still need this change?

It seems like the default is "Major", now.

Reply:
@gearama in case you missed this:
https://github.com/Azure/azure-sdk-for-cpp/pull/5681#discussion_r1638525904

> With [Azure/azure-sdk-tools#8373](https://github.com/Azure/azure-sdk-tools/pull/8373) and the sync #5699 merged, do we still need this change?
> 
> It seems like the default is "Major", now.

Reply:
yes, we still need it , the changes made in the test proxy need to be brought inoto our repo , anbd i don't want to pack it in with this change.